app/helpers/avo/resources_helper.rb

```diff
@@ -66,5 +66,37 @@ def item_select_all_input
           tippy: "tooltip",
         }
     end
+
+    def resource_location_markers(resources)
+      return [] unless resources.first.record.respond_to? :coordinates
+
+      resources.map do |resource|
+        resource_location_marker(resource)
+      end.compact
+    end
+
+    def resource_map_options(resource, map_type: :index)
+      options_method = "resource_#{map_type}_map_options".to_sym
+      if resource.respond_to?(options_method)
```

Comment:
For the map marker configuration and the map options configuration, right now I'm using method defined on the Resource class. Would it be more conventional to define those another way, either as class methods (like `search_query`) or via a new `location_marker` DSL call?

Reply:
These configuration points were moved to elements of a hash class attr.